packages/engine/src/scene/materials/systems/MaterialLibrarySystem.tsx

```diff
@@ -89,7 +89,7 @@ const MaterialEntityReactor = () => {
   const entity = useEntityContext()
   const materialComponent = useComponent(entity, MaterialStateComponent)
   useEffect(() => {
-    if (!materialComponent.instances.value!) return
+    if (!materialComponent.instances.value! || !SourceComponent.entitiesBySource) return
```

Comment:
entitiesBySource is an object and will always return as truthy, I think the correct fix here is to check for the record read on line 94 prior to the nested for loop

Reply:
oh, github was showing me out of date info lol...